homeassistant/components/here_travel_time/sensor.py

```diff
@@ -175,21 +176,29 @@ async def async_setup_platform(
         destination = None
         destination_entity_id = config[CONF_DESTINATION_ENTITY_ID]
 
-    travel_mode = config[CONF_MODE]
     traffic_mode = config[CONF_TRAFFIC_MODE]
```

Comment:
Is there a particular rationale why `traffic_mode` is not a part of `HERETravelTimeConfig`? It naively seemed similar to the other configuration options. I was thinking it might be because the plan is to create a separate sensor for both, but i could see that applying to others like `route_mode` too.

Reply:
The motivation for the question was around simplification related to the `HERETravelTimeConfig` comment.

Reply:
It is currently separate because it is not needed by the coordinator to communicate with the HERE api. Its only use is to define if DURATION or DURATION_IN_TRAFFIC should be used for the native_value of this sensor.

I would therefor argue that it doesn't really belong in `HERETravelTimeConfig`.

The issue will be resolved when all atributes are split into separate sensors in the future.
This is why I would ignore this issue for now.

Reply:
OK sounds good. I am happy to review those follow up PRs if you want to tag me. I think that sounds like a nice change.